### DISCUSSION TITLE
Tweaks are for current user or global?

### DISCUSSION THREAD

**Discussion by aletz9** [ChrisTitusTech/winutil#696] [Q&A]
I know the script are executed as admin, just wondering if i execute the tweaks to disable telemetry, etc. 
If later i create another user, it will be the same for that user, right?

Thanks.

**Comment by DeveloperDurp** [comment]
depends on the tweak, some modifications can only be done at the user level (think personalization) otherwise things are done at the system level.

**Comment by anonyao** [comment]
> I know the script are executed as admin, just wondering if i execute the tweaks to disable telemetry, etc. 
> 
> If later i create another user, it will be the same for that user, right?
> 
> 
> 
> Thanks. 

Broo disable it why should u even have it ? If u gameing it only take performance and send your info to ms?
